great_expectations/util.py

```diff
@@ -1,6 +1,73 @@
 import hashlib
-import numpy.testing as testing
-from numpy import nan as nan
+import pandas as pd
```

Comment:
Does this fully rationalize our `util` files?
IIR, we had utility functions in ge.__init__.py, ge.util.py, ge.dataset.__init__.py, and ge.dataset.util.py.

Reply:
Good point. I don't think it full does yet. It does get everything out of __init__ files, but there is, I think, still some confusion about what should go into great_expectations.dataset.util and great_expectations.util.

I'll do another move as part of this PR to get that cleaner.

Reply:
Upon looking at this more closely, I think that belongs in a separate PR (specifically because of things like DotDict being declared in both places and continued ambiguity in my mind regarding the right place for helpers like "from_pandas"--currently those are top-level things but they may belong better in dataset.utils. I'm going to proceed with the as-approved version.